keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -767,24 +830,37 @@ enhancement:
       - kube-scheduler
 
 * **Does enabling the feature change any default behavior?**
-  No changes for existing storage driver deployments.
+
+  Enabling it only in kube-scheduler and api-server and not any of the
+  running CSI drivers causes no changes. Everything continues as
+  before because no `CSIStorageCapacity` objects are created and
+  kube-scheduler does not wait for any.
+
+  That changes once the feature is enabled in a CSI driver. Then pod
+  scheduling becomes more likely to pick suitable nodes. This happens
+  automatically, without having to change application deployments.
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
   Yes.
 
-  When disabling support in the API server, the new object type and
-  thus all objects will disappear together with the new flag in
-  `CSIDriver`, which will then cause kube-scheduler to revert to
-  provisioning without capacity information. However, the new objects
-  will still remain in the etcd database.
+  In Kubernetes 1.19 and 1.20, registration of the
+  `CSIStorageCapacity` type was controlled by the feature gate. In
+  1.21, the type will always be enabled in the v1beta1 API
+  group. Depending on the combination of Kubernetes release and
+  feature gate, the type will be disabled. However, any existing
+  objects will still remain in the etcd database, they just won't be
+  visible.
 
-  When disabling it in the scheduler, provisioning directly reverts
-  to the previous behavior.
+  When the type is disabled, external-provisioner will be unable to update
+  objects: this needs to be treated with exponential backoff just like other
+  communication issues with the API server.
 
-  External-provisioner will be unable to update objects: this needs to
-  be treated with exponential backoff just like other communication
-  issues with the API server.
+  The new flag in `CSIDriver` will be preserved when disabling the
+  feature gate or when rolling back to a release older than 1.19. kube-scheduler
```

Comment:
on rollback it won't be preserved, because CSIDriver type will not have that field in older release.

So it's only preserved on feature gate disablement.

Reply:
Fixed.